docs/bugpattern/CheckedExceptionNotThrown.md

```diff
@@ -18,6 +18,4 @@ Response handle(Request request) {
 }
 ```
 
-Including unthrown exceptions can be reasonable where the method is overridable,
-as overriding methods will not be able to declare that they throw any exceptions
-not included.
+The check will not flag overridable methods.
```

Comment:
I think the previous text was clearer in that it described **why** that would be the case ("it can be reasonable"). I'd rather **add** to the previous text than replace it.
```suggestion
Including unthrown exceptions can be reasonable where the method is overridable,
as overriding methods will not be able to declare that they throw any exceptions
not included. The check will therefore not flag overridable methods.
```

Reply:
Moved up and parenthesized. The last sentence describes the scope of the check, as a kind of disclaimer. Once the check is adapted to inspect overridable methods it might say "The check will flag non-overridable methods and methods that declare throwing a checked exception where none of the overridden methods throw that exception."